src/api/render-function.md

```diff
@@ -141,23 +141,23 @@ Merge multiple props objects with special handling for certain props.
 
 ## cloneVNode() {#clonevnode}
 
-Clones a vnode.
+Clona un vnode.
 
-- **Type**
+- **Tipo**
 
   ```ts
   function cloneVNode(vnode: VNode, extraProps?: object): VNode
   ```
 
-- **Details**
+- **Dettagli**
 
-  Returns a cloned vnode, optionally with extra props to merge with the original.
+  Restituisce un vnode clonato, opzionalmente con proprietà aggiuntive da unire all'originale.
```

Comment:
qui si riferisce a "extraProps" nella funzione.
Nel file del Glossario raccomandano di usare prop(s) cercando di evitare di estenderlo in properties se non esplicitamente scritto.
> prop non va tradotto, properties si.

quindi tradurrei:
Restituisce un vnode clonato, con extraProps opzionali da unire all'originale.

vedi anche righe successive, 156 - 158